lib/administrate/engine.rb

```diff
@@ -19,5 +18,7 @@
 module Administrate
   class Engine < ::Rails::Engine
     isolate_namespace Administrate
+
+    Engine.config.assets.precompile << /\.(?:svg)\z/
```

Comment:
Looks like newer rails-sprocket gem requires to gems/engines to be explicit about assets. Without this Sprockets 3 crashes with:

```
Sprockets::Rails::Helper::AssetNotPrecompiled

Asset was not declared to be precompiled in production.
Add `Rails.application.config.assets.precompile += %w( administrate/search.svg )` to `config/initializers/assets.rb` and restart your server
```